src/lib/compile-nools-rules.js

```diff
@@ -25,10 +25,14 @@ const compileNoolsRules = async (projectDir, options = {}) => {
       throw new Error(`Both legacy and declarative files found. You should either have rules.nools.js xor ${DECLARATIVE_NOOLS_FILES} files.`);
```

Comment:
Might I suggest we change this file name to exclude "nools" term? `compile-rules` ?

Reply:
I've renamed to `compile-rules.js`. As long as I'm moving files around - I've also moved all compilation related code into folders called `compilation` (one for src and one for test).